Graffle.FlowSdk/Types/StructType.cs

```diff
@@ -0,0 +1,86 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Graffle.FlowSdk.Types
+{
+    public class StructType : FlowValueType
+    {
+        protected StructType() { }
+
+        public StructType(string id, List<(string name, FlowValueType value)> fields)
+        {
+            Data = (id, fields);
+        }
+
+        public StructType((string id, List<(string name, FlowValueType value)> fields) value)
+        {
+            Data = value;
+        }
+
+        public string Id => Data.id;
+
+        public List<(string name, FlowValueType value)> Fields => Data.fields;
+
+        public (string id, List<(string name, FlowValueType value)> fields) Data { get; set; }
```

Comment:
Any feelings on this usage of value tuples? It doesn't look great but its probably the cleanest way to keep this all together